src/openforms/registrations/contrib/zgw_apis/checks.py

```diff
@@ -0,0 +1,56 @@
+from django.utils.translation import gettext_lazy as _
+
+from requests.models import HTTPError
+from zds_client.client import ClientError
+
+from openforms.plugins.exceptions import InvalidPluginConfiguration
+from openforms.registrations.contrib.zgw_apis.models import ZgwConfig
+
+
+def check_config():
+    config = ZgwConfig.get_solo()
+
+    # We need to check 3 API's for this configuration.
+    services = (
+        (
+            "zrc_service",
+            "Zaken API",
+            "zaak",
+        ),
+        (
+            "drc_service",
+            "Documenten API",
+            "enkelvoudiginformatieobject",
+        ),
+        (
+            "ztc_service",
+            "Catalogi API",
+            "zaaktype",
+        ),
+    )
+
+    for service_field, api_name, api_operation in services:
```

Comment:
```suggestion
    for service_field, api_name, api_resource in services:
```
because `operation = resource + verb`